api/src/main/scala/WebServer.scala

```diff
@@ -0,0 +1,125 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import akka.http.scaladsl.unmarshalling.Unmarshaller._
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import DefaultJsonProtocol._
```

Comment:
Interesting. This seems to be required to be able to call `toJson` on the `output` below -- without it, the project won't compile on my local because

```scala
Cannot find JsonWriter or JsonFormat type class for scala.collection.immutable.Map[String,Int]
```